### PR TITLE
[BEAM-2517] Add instructions on how to build and use the Python tarball.

### PR DESCRIPTION
**Pull request by udim**

### CODE REVIEW

**Comment by udim** [comment]
R: @melap

File: src/contribute/contribution-guide.md [hunk 1]
```diff
@@ -218,6 +218,26 @@ push. You'll run:
 
 	$ git push <GitHub_user> <my-branch> --force
 
+### Building
+
+#### Python SDK
+
+Before testing SDK code changes remotely, the Beam tarball has to be built. From
```
Comment (melap) [inline src/contribute/contribution-guide.md]: "the Beam tarball has to be built." -> "you must build the Beam tarball."

Comment (udim) [reply src/contribute/contribution-guide.md]: done

**Comment by melap** [comment]
@asfgit merge

**Comment by asfgit** [comment]
Error: Timed out trying to find verification job. Check Jenkins (https://builds.apache.org/job/beam_PreCommit_Website_Merge/) to ensure job is configured correctly.

**Comment by melap** [comment]
Testing queue is long right now, will retry merge a bit later

**Comment by melap** [comment]
@asfgit merge

**Pull request closed without merging**